src/app/src/main/kotlin/org/vaccineimpact/api/app/models/ChunkedFile.kt

```diff
@@ -0,0 +1,34 @@
+package org.vaccineimpact.api.app.models
+
+import org.vaccineimpact.api.app.ChunkedFileManager.Companion.UPLOAD_DIR
+import java.io.File
+import java.util.concurrent.ConcurrentHashMap
+
+data class ChunkedFile(val totalChunks: Int,
+                       val totalSize: Long,
+                       val chunkSize: Long,
+                       val uniqueIdentifier: String,
+                       val originalFileName: String
```

Comment:
Don't need this parameter?

Reply:
We use it (along with all the others) for checking consistency across POSTs...